app/script/util/URLUtil.js

```diff
@@ -75,7 +75,6 @@ z.util.URLUtil = (() => {
    */
   const _getDomainName = (url = '') => {
     return url
-      .toLowerCase()
```

Comment:
This function is only used to display the link preview url. So, unless I am missing something very important, I guess it's pretty safe to remove that lowercasing here...
@gregor @bennyn do you know the reason for lowercasing this?

Reply:
If tests are not failing, you are good to go! 🚦

Reply:
The lowercasing was placed to catch cases like WWW.WIRE.COM and make sure we show wire.com instead. As tlds are case insensitive we should handle this properly and lowercase that part of the url.